src/App.js

```diff
@@ -1,28 +1,13 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
+import 'bootstrap/dist/css/bootstrap.min.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <Calculator />
+      {' '}
     </div>
   );
 }
```

Comment:
- [ ] Great job you have imported the Calculator.js in the App.js file, however, The App.js component should be class-based.
-------------------------------------- Yours 

![storage2](https://user-images.githubusercontent.com/55635977/133324651-0883c16c-e707-4ab3-97ec-e3713afdc83e.png)

------------------------------------- Components


![storage1](https://user-images.githubusercontent.com/55635977/133318323-e3a5036c-7db4-43dc-a31e-c470edf9bfb6.png)